backend/clickhouse/migrations/000019_create_error_objects_table.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS error_objects (
```

Comment:
as mentioned offline, we'll want a user identifier here as well
should service name and service version be part of the schema as well?
what about custom attributes, could we store these as a map?
<img width="599" alt="Screenshot 2023-08-28 at 11 30 10 AM" src="https://github.com/highlight/highlight/assets/1351531/f29ddf58-ebd0-467d-8637-45e72e8c9b66">

Reply:
I think it would make the most sense to store them in a separate `error_fields` table, I may wait to do something here until we decide to support searching by these fields

Reply:
But yes, I plan on adding service name, version, and client_id to the error object to support those filters